go.mod

```diff
@@ -18,9 +18,9 @@ require (
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.70.1
+	k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf
 	k8s.io/kubectl v0.23.9
 	open-cluster-management.io/addon-framework v0.2.0
-	open-cluster-management.io/governance-policy-propagator v0.8.0
```

Comment:
Huh. It's interesting this isn't needed here. I don't recall why it was here--did `go mod tidy` strip it out?

Reply:
I didn't event pay attention, good catch! `go mod tidy` must have stripped it out, but I thought we had that in a CI check? Am I crazy?